trio/tests/test_exports.py

```diff
@@ -96,6 +101,30 @@ def no_underscores(symbols):
         script = jedi.Script(f"import {modname}; {modname}.")
         completions = script.complete()
         static_names = no_underscores(c.name for c in completions)
+    elif tool == "mypy":
+        if not RUN_SLOW:
+            pytest.skip("use --run-slow to check against mypy")
```

Comment:
Does `pytest.skip` not need a return? I just want to be sure!

Reply:
Looks like it's marked `NoReturn` and works by raising a special exception: https://docs.pytest.org/en/stable/_modules/_pytest/outcomes.html#skip
and definitely worked on my machine!

Though as the doc says should maybe use `pytest.mark.skipif` instead

Reply:
oh nvm, cant use `pytest.mark.skipif` on a parametrized variable.